plugins/adrv9002.c

```diff
@@ -514,6 +520,38 @@ static void adrv9002_check_channel_status(struct plugin_private *priv,
 	}
 }
 
+static void adrv9002_check_nco_freq_support(struct plugin_private *priv, const int channel,
+					    const bool tx)
+{
+	int ret;
+	long long dummy;
+	char gtk_str[32], label_str[32];
+	struct iio_widget *nco;
+
+	if (tx) {
+		nco = &priv->tx_widgets[channel].nco_freq;
+		sprintf(gtk_str, "nco_freq_rx%d", channel + 1);
```

Comment:
Not sure, but should string "nco_freq_rx" have tx instead of rx?

Reply:
Yeps. It works because on the other branch I'm actually doing tx :facepalm: . Will do a v2...